docs/uns-network-guides/docker-compose-management.md

```diff
@@ -2,7 +2,11 @@
 
 ## Main management commands
 
-Coming soon...
+- get UNS Core version from docker image:
+
+```
+docker image inspect --format="{{ index .Config.Labels \"org.label-schema.version\"}}" universalnamesystem/core:[sandbox | livenet]
```

Comment:
```suggestion
docker image inspect --format="{{ index .Config.Labels \"org.label-schema.version\"}}" universalnamesystem/core:livenet
```
And you can add:

"Replace `livenet` by `sandbox` for Sandbox images 😉" (without ")